pkg/ibmvpc/parser.go

```diff
@@ -155,9 +155,7 @@ func VPCConfigsFromResources(rc *datamodel.ResourcesContainerModel, vpcID, resou
 		return nil, err
 	}
 
-	if debug {
-		printVPCConfigs(res)
-	}
+	printVPCConfigs(res)
```

Comment:
We are making here tons of calls to `logging.Debugf()` which will do nothing if verbosity is not set to high. How about adding a method to the logger to get its verbosity level, and only call `printVPCConfigs()` if the verbosity level is high?

Reply:
fixed